packages/main-ui/src/pages/components/community.rs

```diff
@@ -44,7 +44,9 @@ pub fn ComingSoon(
         div {..attributes,
             div { class: "w-full h-full bg-bg flex flex-col items-center justify-center rounded-[20px] gap-30",
                 icons::ComingSoon {}
-                p { class: "text-5xl font-bold text-text-primary", "Coming soon" }
+                p { class: "text-5xl font-bold text-text-primary max-[900px]:text-[24px]",
```

Comment:
```suggestion
                p { class: "text-5xl font-bold text-text-primary max-[900px]:text-2xl",
```
https://tailwindcss.com/docs/font-size
확인하셔서 있는 것들은 가급적이면 있는걸로 써주세요.